### PR TITLE
Make troubleshooting script work on non windows

### PR DESCRIPTION
**Pull request by timja**
In case you're interested, I needed to make these changes so I could run it from cloud shell / mac

### CODE REVIEW

**Comment by timja** [comment]
@keikhara could someone in your team take a look at this please?

File: Troubleshoot/TroubleshootError.ps1 [hunk 1]
```diff
@@ -50,18 +50,6 @@ $azureRmAks = Get-Module -ListAvailable -Name AzureRM.AKS;
 
 if (($null -eq $azureRmProfileModule) -or ($null -eq $azureRmResourcesModule) -or ($null -eq $azureRmOperationalInsights) -or ($null -eq $azureRmAks)) {
 
-    $currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())
```
Comment (timja) [inline Troubleshoot/TroubleshootError.ps1]: this is the specific bit that fails on mac / cloudshell

Comment (keikhara) [reply Troubleshoot/TroubleshootError.ps1]: @timja Yes we will look into it.

**Comment by timja** [comment]
@bragi92 done

**Pull request merged**